SYCLStream.cpp

```diff
@@ -47,7 +47,15 @@ SYCLStream<T>::SYCLStream(const unsigned int ARRAY_SIZE, const int device_index)
   std::cout << "Driver: " << getDeviceDriver(device_index) << std::endl;
   std::cout << "Reduction kernel config: " << dot_num_groups << " groups of size " << dot_wgsize << std::endl;
 
-  queue = new cl::sycl::queue(dev);
+  queue = new cl::sycl::queue(dev, [&](cl::sycl::exception_list l) {
+      try {
```

Comment:
For consistency with the rest of the code, these opening braces should be on newlines.